api_app/ingestors_manager/migrations/0028_ingestor_config_greedybear.py

```diff
@@ -0,0 +1,332 @@
+            "twitter_handle": "",
+            "discover_from": "other",
+            "task_priority": 7,
+            "is_robot": True,
+        },
+    },
+    "playbooks_choice": ["Popular_IP_Reputation_Services"],
+    "name": "GreedyBear",
+    "description": "Queries feeds which are generated by the [GreedyBear Project](https://intelowlproject.github.io/docs/GreedyBear/Introduction/).",
+    "disabled": True,
+    "soft_time_limit": 60,
+    "routing_key": "ingestor",
+    "health_check_status": True,
+    "maximum_jobs": 50,
+    "delay": "00:00:00",
+    "model": "ingestors_manager.IngestorConfig",
+}
+
+params = [
+    {
+        "python_module": {
+            "module": "greedybear.GreedyBear",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "url",
+        "type": "str",
+        "description": "API endpoint",
+        "is_secret": False,
+        "required": False,
+    },
+    {
+        "python_module": {
+            "module": "greedybear.GreedyBear",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "limit",
+        "type": "int",
+        "description": "Max number of results.",
+        "is_secret": False,
+        "required": False,
+    },
+    {
+        "python_module": {
+            "module": "greedybear.GreedyBear",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "feed_type",
+        "type": "str",
+        "description": "The available feed types are log4j, cowrie, and all.",
+        "is_secret": False,
+        "required": False,
+    },
+    {
+        "python_module": {
+            "module": "greedybear.GreedyBear",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "attack_type",
+        "type": "str",
+        "description": "The available attack_type are scanner, payload_request, and all.",
+        "is_secret": False,
+        "required": False,
+    },
+    {
+        "python_module": {
+            "module": "greedybear.GreedyBear",
+            "base_path": "api_app.ingestors_manager.ingestors",
+        },
+        "name": "age",
+        "type": "str",
+        "description": "The available age are recent and persistent.",
+        "is_secret": False,
+        "required": False,
+    },
+]
+
+values = [
+    {
+        "parameter": {
+            "python_module": {
+                "module": "greedybear.GreedyBear",
+                "base_path": "api_app.ingestors_manager.ingestors",
+            },
+            "name": "url",
+            "type": "str",
+            "description": "API endpoint",
+            "is_secret": False,
+            "required": False,
+        },
+        "analyzer_config": None,
+        "connector_config": None,
+        "visualizer_config": None,
+        "ingestor_config": "GreedyBear",
+        "pivot_config": None,
+        "for_organization": False,
+        "value": "greedybear.honeynet.org",
```

Comment:
this needs to contain the protocol

```suggestion
        "value": "https://greedybear.honeynet.org",
```

and then you need to remove it from the python code. 
Why? `url` is a special parameter. It can be used to automatically trigger health checks on the instance. So it needs the protocol

Reply:
Got it! I have updated it as suggested.